.github/workflows/build.yml

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
```suggestion
---
name: ci
```
I would suggest also renaming the file to `ci.yml` for consistency across our repos

Reply:
Done, also gave names to every other workflow/action/step